test/integration/scheduler_perf/dra/templates/resourceclaim-with-selector.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: resource.k8s.io/v1alpha3
+kind: ResourceClaim
+metadata:
+  name: test-claim-with-selector-{{.Index}}
+spec:
+  devices:
+    requests:
+    - name: req-0
+      deviceClassName: test-class
+      selectors:
+      - cel:
+          expression: |-
+            device.capacity['test-driver.cdi.k8s.io'].counters.compareTo(quantity('2')) >= 0 &&
+            device.attributes['test-driver.cdi.k8s.io'].preallocate == true
```

Comment:
Nit: newline at end of file, please.

https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline